manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml

```diff
@@ -0,0 +1,33 @@
+PackageIdentifier: NetSarangComputer.PortX
+PackageVersion: 1.1.3
+Publisher: NetSarangComputer
```

Comment:
```suggestion
Publisher: Support
```

Reply:
Why do we have to change the publishing company?

Reply:
To match Control Panel as this is what Windows Package Manager uses to map the application in `winget list`

If you would like to change the Publisher to something else, please update your application with the changes you would like to make.

Reply:
Oh, thank you. I'll change it to Support and upload it.

Reply:
Changed and committed.